### ISSUE TITLE
Locked out

### ISSUE THREAD

**Issue by loranger** [keybase/keybase-issues#1886]
I received an mail telling my coinbase prove is broken, so I decided to fix it.
I wanted to `keybase prove coinbase` but I wasn't logged in.

So I tried to, but now it feels like I can't do anything :

```bash
$ keybase login
How would you like to sign this install of Keybase?

(1) Use an existing device
(2) Use a paper key
(3) Use my Keybase passphrase
(4) Use GPG
Choose a signing option: 3
Your keybase username: loranger
▶ ERROR no device id set
$ keybase device add
▶ ERROR Login required
```
Do you have any clue on what is the solution in order to get my account back ?

**Comment by maxtaco** [comment]
I think we've fixed this issue, but please let us know what your version is: `keybase version`. Thanks!

**Comment by loranger** [comment]
Already ? Cool !
I use 1.0.2-0, both for client and service.

**Comment by maxtaco** [comment]
Yeah, we fixed this in 1.0.3, have a look!

**Comment by loranger** [comment]
Works perfectly, Thanks !
